src/containers/backpack.jsx

```diff
@@ -107,7 +107,10 @@ class Backpack extends React.Component {
                     username: this.props.username,
                     ...payload
                 }))
-                .then(this.refreshContents);
+                .then(this.refreshContents)
+                .catch(() => {
```

Comment:
It doesn't look like we're logging any of these errors. We probably should, so if/when things start going wrong in the wild, we'll be able to debug remotely.